pro-planner-client/src/routes/LandingPage.jsx

```diff
@@ -1,7 +1,148 @@
+	};
+
+	const isValidCode = value => {
+		return true;
+	};
+
+	const isValidURL = value => {
+		return true;
+	};
+
+	const renderComponent = () => {
+		switch (activeComponent) {
+			case 'Trip':
+				return <Trip />;
+			case 'Outing':
+				return <Outing />;
+			case 'ExistingPlan':
+				if (!isValidInput(inputValue)) {
+					alert(
+						'Enter a URL or Code please! Otherwise, choose to create a plan from scratch.'
+					);
+				} else {
+					return <ExistingPlan />;
+				}
+			default:
+				return (
+					<Container>
+						<Row className="vh-100 d-flex justify-content-center align-items-center">
+							<Col md={8} lg={6} xs={12}>
+								<div className="border border-3 border-primary"></div>
+								<Card className="shadow">
+									<Card.Body>
+										<div className="mt-md-4">
+											<h2 className="fw-bold text-uppercase d-flex justify-content-center align-items-center">
+												ProPlanner
+											</h2>
+											<p className="d-flex justify-content-center align-items-center">
+												Choose the Type of Plan you'd like to Create
+											</p>
+											<div className="mb-3">
+												<Row>
+													<Button
+														variant="primary"
+														size="lg"
+														className="mb-2 createplan-btn"
+														onClick={() => handleButtonClick('Trip')}
+													>
+														Trip
+														<OverlayTrigger
+															placement="top"
+															overlay={renderTooltip('Tooltip for Trip')}
+														>
+															<span className="i-icon">
+																<FiInfo />
+															</span>
+														</OverlayTrigger>
+													</Button>
+												</Row>
+												<Row>
+													<Button
+														variant="primary"
+														size="lg"
+														className="createplan-btn"
+														onClick={() => handleButtonClick('Outing')}
+													>
+														Outing
+														<OverlayTrigger
+															placement="top"
+															overlay={renderTooltip('Tooltip for Outing')}
+														>
+															<span className="i-icon">
+																<FiInfo />
+															</span>
+														</OverlayTrigger>
+													</Button>
+												</Row>
+												<div className="mt-3">
+													<p className="text-center">
+														Joining a Plan? Enter the URL or Code.
+														<Form className="join-form">
+															<Form.Control
+																placeholder="Enter URL/Code"
+																value={inputValue}
+																onChange={handleInput}
+															/>
+															<div className="join-btn">
+																<Button
+																	variant="primary"
+																	type="submit"
+																	className="mt-2"
+																	onClick={() =>
+																		handleButtonClick('ExistingPlan')
+																	}
+																>
+																	Join
+																</Button>
+															</div>
+														</Form>
+													</p>
```

Comment:
I believe this paragraph tag should be placed higher. If not, this needs to be fixed, divs and forms can't be children to paragraph elements.

Reply:
I think we can get rid of the paragraph tags?

Reply:
yeah, either move the closing tag up or remove it, whichever works.